src/resources/js/frontend-ticket-form.js

```diff
@@ -0,0 +1,260 @@
+var tribe_tickets_ticket_form = {};
+
+/**
+ * Provides global stock handling for frontend ticket forms.
+ *
+ * @var object tribe_tickets_stock_data
+ */
+( function( $, my ) {
+	var $tickets_lists   = $( '.tribe-events-tickets' );
+	var $quantity_fields = $tickets_lists.find( '.quantity' ).find( 'input' );
```

Comment:
Wrap these two lines (along with [this one](https://github.com/moderntribe/event-tickets/pull/113/files#diff-0f98cb8f6ef0e2fe89036c6177442ce9R258)) in a `my.init()` to make sure they aren't hunted for in the DOM before they exist on the page (if someone sets a this JS file as a dependency to another that loads in the header or something).

Then you can call it on document ready: `$( function() { my.init(); } );`